tests/checkpatch_wrapper_test.sh

```diff
@@ -42,31 +59,73 @@ function oneTimeTearDown
   rm -rf "$FAKE_KERNEL"
 }
 
-function checkpatch
-{
-  res=$(execute_checkpatch "$FAKE_KERNEL/samples/codestyle_$1.c" 2>&1)
-  [[ "$res" == *"${!MSG[$1]}" ]]
-  assertTrue "Checkpatch should output: ${!MSG[$1]}" $?
-}
-
-function testWarning
+function warning_Test
 {
   checkpatch "warning"
 }
 
-function testError
+function error_Test
 {
   checkpatch "error"
 }
 
-function testChecks
+function checks_Test
 {
   checkpatch "check"
 }
 
-function testCorrect
+function correct_Test
 {
   checkpatch "correct"
 }
 
+function invalid_path_Test
+{
+  local build_fake_path
+  local output
+  local ret
+
+  build_fake_path="$RANDOM/$RANDOM/$RANDOM/xptolala"
+
+  output=$(execute_checkpatch "$build_fake_path")
+  ret="$?"
+
+  [[ "$ret" != 22 ]] && fail "We forced an invalid path and we expect an error"
```

Comment:
This test fails in my machine. Probably it is because of my shunit version, which interprets tests returning false as failed. As a solution, you could add a line with `true` in the end. Or, to avoid this hacky solution, refactor the above to:
```
[[ "$ret" == 22 ]]; assertTrue "We forced an invalid path and we expect an error" $?
```